### PR TITLE
basic test for v1

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

File: testv1.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source common.sh
+
+function test_init_project {
+  header_text "performing init project"
+  kubebuilder init --project-version=v1 --domain example.com <<< "y"
+}
```
Comment (droot) [inline testv1.sh]: Don't we want to run `make` here like we are doing in `test_init_project_manual_dep_ensure` ?

File: testv1.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  make
+}
+
+function test_create_api_controller {
+  header_text "performing creating api and controller"
+  kubebuilder create api --group insect --version v1beta1 --kind Bee <<EOF
+y
+y
+EOF
+}
```
Comment (droot) [inline testv1.sh]: same question as above.

File: testv1.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+function test_create_api_only {
+  header_text "performing creating api only"
+  kubebuilder create api --group insect --version v1beta1 --kind Bee <<EOF
+y
+n
+EOF
+}
```
Comment (droot) [inline testv1.sh]: same q as above.

File: testv1.sh [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+test_init_project
+
+prepare_testdir_under_gopath
+test_init_project
+test_create_api_controller
+
+prepare_testdir_under_gopath
+test_init_project
+test_create_api_only
+
```
Comment (droot) [inline testv1.sh]: don't see test with manual_dep_ensure being invoked here ?

File: testv1.sh (hunk 2 above)
Comment (mengqiy) [reply testv1.sh]: `kubebuilder create api` will automatically invoke `make`.

File: testv1.sh (hunk 1 above)
Comment (mengqiy) [reply testv1.sh]: Added

File: testv1.sh (hunk 4 above)
Comment (mengqiy) [reply testv1.sh]: Added

**Pull request merged**